README.md

```diff
@@ -40,74 +39,11 @@ Our benchmarking method is based on [HelloBench](https://github.com/Tintri/hello
 
 Stargz Snapshotter is a **non-core** sub-project of containerd.
 
-## Quick Start with Kubernetes
+## Install
 
-- For more details about stargz snapshotter plugin and its configuration, refer to [Containerd Stargz Snapshotter Plugin Overview](/docs/overview.md).
+[[☸**Quick Start (Kubernetes)**]](./docs/INSTALL.md#quick-start-with-kubernetes)
 
-For using stargz snapshotter on kubernetes nodes, you need the following configuration to containerd as well as run stargz snapshotter daemon on the node.
-We assume that you are using containerd (> v1.4.2) as a CRI runtime.
-
-```toml
-version = 2
-
-# Plug stargz snapshotter into containerd
```

Comment:
This sections shouldn't be moved from README to INSTALL.md.
Please remain this section in README and just add a link something like the following.

----

 - For more details about installing Stargz Snapshotter, refer to [Install Stargz Snapshotter](/docs/INSTALL.md).

Reply:
I think Systemd is on an equality with Kubernetes. Why not put them together?  The project only wants to support Kuberntes?

Reply:
> Why not put them together?

Because "Quick Start with Kubernetes" in README is not the installation guide for kuberentes. I agree that we should add an installation guide for Kubernetes to INSTALL.md. But that section is literary a "Quick Start" guide and the contents are not enough (e.g. that section doesn't tell you how to configure your GKE cluster with Stargz Snapshotter). We need a more detailed step-by-step guide for each popular services/distros. For example, https://ricardorocha.io/blog/gke-custom-containerd/ is a good guide for installing Stargz Snapshotter to Kubernetes on GKE. We want something similar in our INSTALL.md (for other platforms as well, not limited to GKE).

>  Does this project only support Kuberntes?

No, this doesn't. We are aiming at tools other than Kubernetes as well (Please see also https://github.com/containerd/stargz-snapshotter/issues/258)

Reply:
You are right.It makes sense. I have modify according to comments